app/views/bgeigie_imports/_static_metadata.html.erb

```diff
@@ -1,28 +1,14 @@
 <div class="span2">
   <dl>
-    <dt>
-      <%= t('.uploaded_by') %>
-    </dt>
-    <dd>
-      <%= @bgeigie_import.user.name_or_email %>
-    </dd>
-    <dt>
-      <%= t('.filename') %>
-    </dt>
-    <dd>
-      <%= @bgeigie_import.source.filename %>
-    </dd>
-    <dt>
-      <%= t('.number_of_lines') %>
-    </dt>
-    <dd>
-      <%= @bgeigie_import.lines_count %>
-    </dd>
-    <dt>
-      <%= t('.number_of_measurements') %>
-    </dt>
-    <dd>
-      <%= @bgeigie_import.processed? ? @bgeigie_import.bgeigie_logs.size : '-' %>
-    </dd>
+    <dt><%= t('.uploaded_by') %></dt>
+    <dd><%= @bgeigie_import.user.name_or_email %></dd>
+    <dt><%= t('.filename') %></dt>
+    <dd><%= @bgeigie_import.source.filename %></dd>
+    <dt><%= t('.number_of_lines') %></dt>
+    <dd><%= @bgeigie_import.lines_count %></dd>
+    <dt><%= t('.number_of_measurements') %></dt>
+    <dd><%= @bgeigie_import.processed? ? @bgeigie_import.bgeigie_logs.size : '-' %></dd>
+    <dt>Highest Measurement</dt>
```

Comment:
Seems like we might not want to break the translation pattern here. Mind at least dropping it into the `en` strings file?

Reply:
Yep, done.